commands/host.vegman

```diff
@@ -224,6 +224,90 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to default on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
```

Comment:
I suggest to use `busctl` here too, as it would be a shorter way.

Reply:
`ipmitool` here also clears the other boot device settings, and I can't say that's bad for the 'reset to defaults' case.

Reply:
Ok, let it be as is.